src/bookshelf/stest/rebar.config

```diff
@@ -20,6 +21,6 @@
 %% markdown can be checked in and will be browsable on github. The
```

Comment:
:+1: let's nuke it.  It would be could if we used edown, but right now we don't.